web/Views/Property/_PropertyTable.cshtml

```diff
@@ -0,0 +1,97 @@
+﻿@model PropertiesDashboardViewModel
+
+@if (Model.Properties.Count == 0)
+{
+    <h2 class="font-h2">No properties found</h2>
+    return;
+}
+<div class="table-responsive mt-4">
+    <table class="table">
+        <thead class="bg-pale-blue">
+        <tr>
+            <th scope="col">Status</th>
+            <th scope="col">Address</th>
+            <th scope="col" style="width: 150px">Type</th>
+            <th scope="col">Bedrooms</th>
+            <th scope="col">Description</th>
+            <th scope="col">Availability</th>
+            <th scope="col">Rent (pcm)</th>
+            <th scope="col">Total units</th>
+            <th scope="col">Available units</th>
+            <th scope="col">Created</th>
+            <th scope="col">Details</th>
+            @if (User.IsInRole("Admin"))
+            {
+                <th scope="col">Landlord</th>
+                <th scope="col">Match</th>
+            }
+        </tr>
+        </thead>
+        <tbody>
+        @foreach (var prop in Model.Properties)
+        {
+            <tr>
+                <td class="text-nowrap">
+                    <partial name="../Property/Details/_AvailabilityPill" model="prop"/>
+                </td>
+                <td>
+                    @prop.Address.AddressLine1,
+                    @prop.Address.AddressLine2@(prop.Address.AddressLine2 != null ? "," : "")

```

Comment:
I think it would be nice to combine these 2, and below for addressline 3 as well :) 
```suggestion
                    @(prop.Address.AddressLine2 == null ? "" : prop.Address.AddressLine2 + ",")

```